xml/docu_styleguide.structure.xml

```diff
@@ -817,12 +891,21 @@ and change the value of &lt;varname&gt;DISPLAYMANAGER&lt;/varname&gt;.</screen>
     <title>Emphasis</title>
     <para>
       Where possible, indicate stress with language only. If that is not
-      possible, use the <tag class="emptytag">emphasis</tag> element to
-      indicate stress.
-    </para>
-    <para>
-      Where added emphasis is needed, use the
-      <tag class="attribute">role="bold"</tag> attribute.
+      possible, <phrase outputformat="db">use the
+      <tag
+      class="emptytag">emphasis</tag>
+      element</phrase><phrase
+      outputformat="adoc">surround the phrase with
+      underscores (<literal>_phrase_</literal>)</phrase> to indicate stress.
+    </para>
+    <para>
+      Where added emphasis is needed, <phrase outputformat="db">use the
+      <tag class="attribute">role="bold"</tag>
+      attribute.</phrase><phrase
+      outputformat="adoc">enclose the phrase in
+      a single pair of asterisks (<literal>*phrase*</literal>). To emphasize
+      characters within a word, use a pair of double asterisks
+      (<literal>char**act**ers</literal>).</phrase>
     </para>
 <screen>This will be displayed in &lt;emphasis&gt;italics&lt;/emphasis&gt;. This
 will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>
```

Comment:
Add missing profiling attribute:

```suggestion
<screen outputformat="db">This will be displayed in &lt;emphasis&gt;italics&lt;/emphasis&gt;. This
will be displayed in &lt;emphasis role="bold"&gt;bold&lt;/emphasis&gt;</screen>

<screen outputformat="adoc"> This will be displayed _italic_
 This will be displayed in *bold*.</screen>
```

Don't forget to check the ADoc variant.